### ISSUE TITLE
suggestion:Use #%% to execute a piece of code like a spyder when press alt+enter

### ISSUE THREAD

**Issue by Louis-udm** [lorenzo2897/vscode-ipe#163]
this will be a lot easier.

**Comment by LukeyQ1** [comment]
I believe you can edited to these whatever keybindings you wish, these are just the defaults.
https://code.visualstudio.com/docs/getstarted/keybindings

**Comment by j9ac9k** [comment]
Just adding that pycharm has the same comment to make chunk separation

**Comment by mwouts** [comment]
The `# %%` marker for cells is indeed common practice. It was introduced by Spyder five years ago (cf. https://github.com/spyder-ide/spyder/issues/7933), and is now supported by PyCharm (professional), vscode with the vscode Jupyter plugin, PTVS, and obviously, Hydrogen. In addition: scripts in that format can be converted to Jupyter notebooks (and back) using Jupytext.

**Comment by AlJohri** [comment]
yeah, this will be a lot easier and then you can get the Run Cell button like https://github.com/DonJayamanne/vscodeJupyter has
